src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
we can remove it

Reply:
https://github.com/mate-academy/react_todo-app-add-and-delete/pull/327/files#r1124831257